### PR TITLE
add faq page

### LINKED ISSUE #274: add Help page and links
As a collection admin, I'd like to be able to access some basic instructions on how to use the site. 

We need a "Help" page which is prominently linked on the collection page. 

The Help page can be organized as FAQs, with linkable anchors so we can use specific URLs when answering support questions.

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

closes #274 

#### What's this PR do?

This adds an FAQ / help page with expandable questions.

#### How should this be manually tested?

If you go to `/help` you should see the page. If you are an admin you should see the admin specific questions, if not you should see the viewer specific ones. You can change the value of `SETTINGS.is_admin` to test out both views.

### CODE REVIEW

File: static/js/actions/commonUi.js [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
-export const SHOW_MENU = qualifiedName('SHOW_MENU');
+export const SHOW_MENU = qualifiedName("SHOW_MENU");
 export const showMenu = createAction(SHOW_MENU);
 
-export const HIDE_MENU = qualifiedName('HIDE_MENU');
+export const HIDE_MENU = qualifiedName("HIDE_MENU");
 export const hideMenu = createAction(HIDE_MENU);
+
+export const SHOW_FAQ_ENTRY = qualifiedName("SET_SHOW_FAQ_ENTRY");
+export const showFAQEntry = createAction(SHOW_FAQ_ENTRY);
```
Comment (gsidebo) [inline static/js/actions/commonUi.js]: Should probably be called `TOGGLE_FAQ_VISIBILITY` and `toggleFAQVisibility` or something like that

File: static/js/containers/HelpPage.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import FAQ from "../components/FAQ";
+import { showFAQEntry } from '../actions/commonUi';
+
+class HelpPage extends React.Component {
+  toggleShowFAQ = R.curry((questionName, e) => {
+    const { dispatch } = this.props;
+
+    e.preventDefault();
+
+    dispatch(showFAQEntry(questionName));
```
Comment (gsidebo) [inline static/js/containers/HelpPage.js]: Nit: can we get rid of the blank line above? (I prefer to drop the blank line after `const` as well but I guess that's a semi-accepted pattern)

File: ui/templates/ui/help.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block title %}OVS | Video Detail{% endblock %}
+{% block pagetitle %}OVS: Video Detail{% endblock %}
+{% block content %}
+
```
Comment (gsidebo) [inline ui/templates/ui/help.html]: Nit: unnecessary blank line

File: ui/views.py [hunk 4]
```diff
@@ -91,6 +91,22 @@ def get_context_data(self, video_key, **kwargs):  # pylint: disable=arguments-di
 
 
 @method_decorator(login_required, name='dispatch')
+class HelpPageView(TemplateView):
```
Comment (gsidebo) [inline ui/views.py]: Nit: we should define this under `VideoEmbed` to keep the collection/video template views defined proximally

File: ui/templates/ui/help.html [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block title %}OVS | Video Detail{% endblock %}
+{% block pagetitle %}OVS: Video Detail{% endblock %}
```
Comment (gsidebo) [inline ui/templates/ui/help.html]: Missed this during the first review. title & pagetitle block text needs to be updated

Comment (aliceriot) [reply ui/templates/ui/help.html]: lol :woman_facepalming:

File: static/js/actions/commonUi.js (hunk 1 above)
Comment (aliceriot) [reply static/js/actions/commonUi.js]: sure, that makes sense

**Comment by aliceriot** [comment]
@gsidebo comments are addressed I think

**Comment by pdpinch** [comment]
Can we get a screenshot of this (or a PR build) so @roberthouse54 @emmello and I can see before we merge?

**Comment by aliceriot** [comment]
faq2
faq

**Comment by emmello** [comment]
Screen shots look good other than my typo of Delay Posting a video - second to last word should be your instead of you.

**Comment by aliceriot** [comment]
Typo should be fixed now

**Pull request merged**